src/components/element-editor/basic-details/basic-details.styles.tsx

```diff
@@ -1,6 +1,11 @@
+import { Color } from "@lifesg/react-design-system/color";
 import { Accordion } from "@lifesg/react-design-system/accordion";
 import styled from "styled-components";
 
+interface IProps {
```

Comment:
since there are multiple components here, the name of the typing will need to be more specific

```suggestion
interface FieldEditorAccordionItemProps {
```